nestkernel/source_table.cpp

```diff
@@ -55,17 +55,15 @@ nest::SourceTable::initialize()
     const thread tid = kernel().vp_manager.get_thread_id();
     sources_[ tid ].resize( 0 );
     resize_sources( tid );
-    is_cleared_[ tid ] = false;
-    saved_entry_point_[ tid ] = false;
```

Comment:
Setting to false here is no longer needed because you now initialize properly on lines 48-49, isn't it? Just want to confirm my understanding of the code.

Reply:
Correct!